code/modules/genetics/mutations/flame_retardent.dm

```diff
@@ -0,0 +1,23 @@
+//Macro: MUTATION_FLAME_RETARDENT
```

Comment:
```suggestion
//Macro: MUTATION_FLAME_RETARDANT
```
I also suggest changing the filename if you want, for the sake of consistency.

Reply:
oh didn't see I spelled it diffrently in one spot. My bad. Good catch.